config.json

```diff
@@ -1,163 +1,52 @@
-          "packageName": "StyleDictionaryColor",
+          "destination": "colors.scss",
+          "format": "scss/variables",
           "filter": {
-            "attributes": {
-              "category": "color"
-            }
+            "type": "color"
           }
         },
         {
-          "destination": "StyleDictionarySize.kt",
-          "format": "compose/object",
-          "className": "StyleDictionarySize",
-          "packageName": "StyleDictionarySize",
-          "type": "float",
-          "filter": {
-            "attributes": {
-              "category": "size"
-            }
-          }
-        }
-      ]
-    },
-    "ios": {
-      "transformGroup": "ios",
-      "buildPath": "build/ios/",
-      "files": [
-        {
-          "destination": "StyleDictionaryColor.h",
-          "format": "ios/colors.h",
-          "className": "StyleDictionaryColor",
-          "type": "StyleDictionaryColorName",
+          "destination": "typography.scss",
+          "format": "scss/variables",
           "filter": {
-            "attributes": {
-              "category": "color"
-            }
-          }
-        },
-        {
-          "destination": "StyleDictionaryColor.m",
-          "format": "ios/colors.m",
-          "className": "StyleDictionaryColor",
-          "type": "StyleDictionaryColorName",
-          "filter": {
-            "attributes": {
-              "category": "color"
-            }
-          }
-        },
-        {
-          "destination": "StyleDictionarySize.h",
-          "format": "ios/static.h",
-          "className": "StyleDictionarySize",
-          "type": "float",
-          "filter": {
-            "attributes": {
-              "category": "size"
-            }
-          }
-        },
-        {
-          "destination": "StyleDictionarySize.m",
-          "format": "ios/static.m",
-          "className": "StyleDictionarySize",
-          "type": "float",
-          "filter": {
-            "attributes": {
-              "category": "size"
-            }
-          }
-        }
-      ]
-    },
-    "ios-swift": {
-      "transformGroup": "ios-swift",
-      "buildPath": "build/ios-swift/",
-      "files": [
-        {
-          "destination": "StyleDictionary+Class.swift",
-          "format": "ios-swift/class.swift",
-          "className": "StyleDictionaryClass",
-          "filter": {}
-        },
-        {
-          "destination": "StyleDictionary+Enum.swift",
-          "format": "ios-swift/enum.swift",
-          "className": "StyleDictionaryEnum",
-          "filter": {}
-        },
-        {
-          "destination": "StyleDictionary+Struct.swift",
-          "format": "ios-swift/any.swift",
-          "className": "StyleDictionaryStruct",
-          "filter": {},
-          "options": {
-            "imports": "SwiftUI",
-            "objectType": "struct",
-            "accessControl": "internal"
+            "type": "typography"
```

Comment:
this is the easy .json way of filtering, there is an option to change config.json into config.js where we can write our own custom filtering functions